decidim-participatory_processes/app/controllers/decidim/participatory_processes/admin/participatory_process_attachment_collections_controller.rb

```diff
@@ -17,6 +18,10 @@ def after_destroy_path
         def collection_for
           current_participatory_process
         end
+
+        private
+
+        def tab_menu_name = :admin_participatory_process_attachments_menu
```

Comment:
Consistency with the rest of names:
```suggestion
        def tab_menu_name = :participatory_process_admin_attachments_menu
```